x-pack/plugins/observability_solution/slo/public/pages/slos/components/common/quick_filters.scss

```diff
@@ -0,0 +1,40 @@
+.controlsWrapper {
+  min-height: $euiSize * 4;
+}
+
+.optionsList--filterBtn, .euiFilterButton {
+  height: $euiFormControlHeight !important;
+}
+
+.euiFilterGroup {
+  height: $euiFormControlHeight !important;
+  border-radius: $euiFormControlBorderRadius !important;
+}
+
+.controlPanel {
+
+  .euiDualRange, .euiRange {
+    height: $euiFormControlHeight !important;
+  }
+
+  .controlPanel--label {
+    height: $euiFormControlHeight !important;
+  }
+}
+
+.dshDashboardViewport-controls {
```

Comment:
It seems like you have a **lot** of unnecessary styles here... For example, `dshDashboardViewport-controls` **never** shows up in the `ControlRenderer` so why are we targeting a dashboard-specific class? We should clean these styles up to **only include the bare minimum**

Reply:
Ok! I was going through and reverting the changes made in https://github.com/elastic/kibana/pull/190636 to avoid any whonkiness in SLOs but I'll clean it up 👍🏻